lib/json_spec/matchers/be_json_eql.rb

```diff
@@ -42,6 +43,11 @@ def including(*keys)
         self
       end
 
+      def order_indifferent(state = true)
```

Comment:
Thoughts about a more human-friendly name for this like `in_any_order`?

```ruby
expect(foo).to be_json_eql(bar).in_any_order
```

Just spitballing!